assets/src/edit-story/app/story/effects/useLoadStory.js

```diff
@@ -89,7 +90,9 @@ function useLoadStory( {
 				};
 
 				// If there are no pages, create empty page.
-				const pages = storyData.length === 0 ? [ createPage() ] : storyData;
+				const storyDataPreMigration = Array.isArray( storyDataRaw ) ? { version: 1, pages: storyDataRaw } : storyDataRaw;
```

Comment:
Could/should this be a migration function as well? I assume no because it's so simple.

Reply:
Great idea. Done.